modules/openshift-architecture-common-terms.adoc

```diff
@@ -0,0 +1,186 @@
+
+cluster Version Operator (CVO)::
+An Operator that checks with the {product-title} Update Service to see the valid updates and update paths based on current component versions and information in the graph.
+
+compute nodes::
+Nodes that are responsible for executing workloads for cluster users. Also known as worker nodes.
+
+configuration drift::
+A situation where the configuration on a node does not match what the machine config specifies.
+
+containers::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+container orchestration engine::
+Software that automates the deployment, management, scaling, and networking of containers.
+
+container workloads::
+Applications that are packaged and deployed in containers.
+
+control groups (cgroups)::
+Partitions sets of processes into groups to manage and limit the resources processes consume.
+
+control plane::
+A container orchestration layer that exposes the API and interfaces to define, deploy, and manage the life cycle of containers. Also known as control plane machines.
+
+CRI-O::
+A Kubernetes native container runtime implementation that integrates with the operating system to deliver an efficient Kubernetes experience.
+
+deployment::
+A Kubernetes resource object that maintains the life cycle of an application.
+
+dockerfile::
+A text file that contains the user commands to perform on a terminal to assemble the image.
+
+hosted control planes::
+A {product-title} feature that enables hosting a control plane on the {product-title} cluster from its data plane and workers. This model performs following actions:
+
+* Optimize infrastructure costs required for the control planes.
+* Improve the cluster creation time.
+* Enable hosting the control plane using the Kubernetes native high level primitives. For example, deployments, stateful sets.
+* Allow a strong network segmentation between the control plane and workloads.
+
+hybrid cloud deployments::
+Deployments that deliver a consistent platform across bare metal, virtual, private, and public cloud environments. This offers speed, agility, and portability.
+
+ignition::
+A utility that {op-system} uses to manipulate disks during initial configuration. It completes common disk tasks, including partitioning disks, formatting partitions, writing files, and configuring users.
+
+installer-provisioned infrastructure::
+The installation program deploys and configures the infrastructure that the cluster runs on.
+
+kubelet::
+A primary node agent that runs on each node in the cluster to ensure that containers are running in a pod.
+
+kubernetes manifest::
+Specifications of a Kubernetes API object in a JSON or YAML format. A configuration file can include deployments, config maps, secrets, daemon sets.
+
+machine Config Daemon (MCD)::
```

Comment:
For MCD and MCO the "Machine" should stay capitalized

Reply:
Addressed